### ISSUE TITLE
Room.State reporting null for rooms Not Enclosed

### ISSUE THREAD

**Issue by david-pbk** [andydandy74/ClockworkForDynamo#256]
**Describe the bug**
Room.State returns null value on Not Enclosed rooms when its boundary elements are deleted in the script.  
I copied Room.States nodes into my script to investigate, and it appears that the area of a room is not updated while this script is run.  The Area > 0 test returns true on all these rooms, while the Room Boundary test returns false, returning a number (110) that has no matching key in the list of keys.
I'm going to try editing this node with a new key and value to see if I can get it to work properly.

**Steps to Reproduce**
Open file with rooms bounded by a linked file
Run script to delete linked file and all rooms that are not Placed.

**Expected behavior**
All rooms that are not Placed should be deleted from the file.

**My setup**
- Operating System: Win 10
- Dynamo version: 1.3.4
- Clockwork version: 1.33.1
- Revit version: 2017.2

**Housekeeping**
Please **make sure these boxes are checked** before submitting your issue:
- [X] I am running a *supported* Clockwork version
- [X] I am running the *latest available* Clockwork version
- [X] My installed version of Clockwork matches the Dynamo version it's installed for
- [X] I do *not* have multiple versions of Clockwork installed for one Dynamo version
- [X] I do *not* have any pre-Clockwork packages installed (category CAAD_RWTH in the node browser)

![image](https://user-images.githubusercontent.com/47759899/52977840-aeecd600-3383-11e9-979e-0927e4cd2abd.png)

**Comment by andydandy74** [comment]
Interesting - I've never used that node downstream in a graph. Makes sense that this error might occur. Can you update this thread once your testing has concluded?

**Comment by david-pbk** [comment]
Adding 110 to the key string and a second Not enclosed to the value list worked for my graph.  I'm using it to grab all the rooms and delete them, but I tested it out by narrowing the selection to each category (Unplaced, Redundant, etc.) and it worked as expected.  I've attached a graph of the modified node.
![capture](https://user-images.githubusercontent.com/47759899/53038344-5e33b680-3431-11e9-8ffa-0b4bad13d37c.png)
